lib/Graph/Graph.cpp

```diff
@@ -820,6 +820,12 @@ ConcatNode *Function::createConcat(llvm::StringRef name,
   return addNode(new ConcatNode(name, OT, ops, dimension));
 }
 
+TileNode *Function::createTile(llvm::StringRef name, NodeValue input,
```

Comment:
similar `asserts` as in other createTile function, or better yet merge these two function in one with optional outTy param to reduce code dups.